Домашка в папке sixth

Kuanysh-kst opened the pull request.

Kuanysh-kst committed:
Create Main.java

Kuanysh-kst committed:
some

Kuanysh-kst committed:
Create .gitignore

blekkiw commented on .gitignore:
```diff
@@ -0,0 +1,7 @@
+
```
Можно поставить просто /out/*

blekkiw commented on src/com/company/hometask/sixth/Measurable.java:
```diff
@@ -0,0 +1,11 @@
+package com.company.hometask.sixth;
+
+public interface Measurable {
```
При чем тут интерфейс?

blekkiw commented on src/com/company/hometask/sixth/ObjectList.java:
```diff
@@ -0,0 +1,35 @@
+    private final int size = 16;
+    private int count = 0;
+
+    private Object[] array = new Object[size];
```
Задача как раз была в том, чтобы сделать массив Measurable. Чтобы потом не делать instanceof и каст

Kuanysh-kst committed:
сделал супер класс Parent

сделал этот класс для того чтобы можно было добавить в массив класса Parent  его наследников

Kuanysh-kst committed:
убрал instentsof
